docs/user/examples.rst

```diff
@@ -0,0 +1,62 @@
+Example projects
+================
+
+* Need inspiration?
+* Want to bootstrap a new documentation project?
+* Want to showcase your own solution?
+
+The following example projects show a rich variety of uses of Read the Docs. You can use them for inspiration, for learning and for recipies to start your own documentation projects. View the *rendered* version of each project and then head over to the Git source to see how it's done and reuse the code.
+
+Sphinx and MkDocs examples
+--------------------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 20 40 20
+
+   * - Topic
+     - Framework
+     - Links
+     - Description
+   * - Basic Sphinx
+     - Sphinx
+     - `[Git] <https://github.com/readthedocs-examples/example-sphinx-basic/>`__ `[Rendered] <https://example-sphinx-basic.readthedocs.io/en/latest/>`__
+     - Basic example of using Sphinx
+   * - Basic MkDocs
+     - MkDocs
+     - `[Git] <https://github.com/readthedocs-examples/example-mkdocs-basic/>`__ `[Rendered] <https://example-mkdocs-basic.readthedocs.io/en/latest/>`__
+     - Basic example of using MkDocs
+
+
+Real-life examples
+------------------
+
+.. image:: _static/images/awesome-list.svg
+  :alt: Awesome List badge
+  :target: https://github.com/readthedocs-examples/
+
+We maintain an **Awesome List** where you can contribute new shiny examples of using Read the Docs. Please refer to the instructions on how to submit new entries on `Awesome Read the Docs Projects <https://github.com/readthedocs-examples/>`_.
+
+
+Contributing an example projects
+--------------------------------
+
+We would love to add more examples that showcase features of Read the Docs or great tools or methods to build documentation projects.
+
+.. admonition:: We **require** that an example project:
```

Comment:
This is an odd syntax we don't use anywhere else. I think we just do a normal list, without the `...` at the front.